### PR TITLE
[`BLIP`] update blip path on slow tests

### PR DESCRIPTION
**Pull request by younesbelkada**
# What does this PR do?

This PR updates the path to the image we use for running BLIP slow tests - as pointed out by @NielsRogge better to upload these images on the Hub in case they get removed from the original place
Happy also to move the Hub repo on `hf-internal-testing` but I am not a member of the org

Also the image is quite large, so I prefer to upload it on the Hub rather than pushing it on the repo here

cc @NielsRogge @sgugger

### CODE REVIEW

**Comment by sgugger** [comment]
Can you add it to hf-internal-testing instead? Might be better there. Happy to merge any PR or approve your demand to join ;-)

**Comment by NielsRogge** [comment]
Thanks for adding! I've transferred it :)

File: tests/models/blip/test_modeling_blip.py [hunk 1]
```diff
@@ -1081,7 +1081,7 @@ def test_model_from_pretrained(self):
 
 # We will verify our results on an image of cute cats
 def prepare_img():
-    url = "https://storage.googleapis.com/sfr-vision-language-research/BLIP/demo.jpg"
+    url = "https://huggingface.co/ybelkada/blip-test-image/resolve/main/demo.jpg"
```
Comment (younesbelkada) [inline tests/models/blip/test_modeling_blip.py:1084]:
```suggestion
    url = "https://huggingface.co/hf-internal-testing/blip-test-image/resolve/main/demo.jpg"
```

**Review by NielsRogge — APPROVED** [review]
Thanks!

**Comment by younesbelkada** [comment]
@sgugger , I think that we can merge this! 🙏

**Review by amyeroberts — APPROVED** [review]
LGTM - thanks for updating!

**Pull request merged**